docs/joss_paper/paper.bib

```diff
@@ -0,0 +1,81 @@
+
+@article{grandgirard2016,
+title = {A 5D gyrokinetic full-f global semi-Lagrangian code for flux-driven ion turbulence simulations},
+journal = {Computer Physics Communications},
+volume = {207},
+pages = {35-68},
+year = {2016},
+issn = {0010-4655},
+doi = {https://doi.org/10.1016/j.cpc.2016.05.007},
+url = {https://www.sciencedirect.com/science/article/pii/S0010465516301230},
+author = {V. Grandgirard and J. Abiteboul and J. Bigot and T. Cartier-Michaud and N. Crouseilles and G. Dif-Pradalier and Ch. Ehrlacher and D. Esteve and X. Garbet and Ph. Ghendrih and G. Latu and M. Mehrenberger and C. Norscini and Ch. Passeron and F. Rozar and Y. Sarazin and E. Sonnendrücker and A. Strugarek and D. Zarzoso},
+}
+
+@misc{ddc,
+    title = {TODO}
+}
```

Comment:
@tpadioleau I am aiming to submit the Gyselalib++ JOSS paper around the 12th May. As the DDC paper has not yet been submitted how do you prefer I reference DDC?

Reply:
Citing submission: ed7e3c59c